test/integration/logic/TypeResolverTest.java

```diff
@@ -73,15 +75,23 @@ public static void close_session() {
         grakn.close();
     }
 
-    private static void define_standard_schema(String fileName) throws IOException {
+    @Before
+    public void setup() {
         transaction = session.transaction(Arguments.Transaction.Type.WRITE);
+    }
+
+    @After
+    public void tearDown() {
+        transaction.close();
+    }
+
+    private static void define_standard_schema(String fileName) throws IOException {
```

Comment:
This should be called in `setup()`

Reply:
ah so turns out half the tests use `define_standard_schema`, and half use `define_custom_schema`, so i will keep it out of `setup()`